### PR TITLE
Reference json

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: src/tests/test_converter.py [hunk 1]
```diff
@@ -22,3 +23,92 @@ def test_apply_filters(self):
...
+            (".", False),
+            ("kdkakdkaf.", False),
+            (".kdljfakld", False),
+            ("", False),
+            ("kdjld_.dkdj_dkldjf", True),
+        )
+
+        for testcase, result in testcases:
+            self.assertEqual(result, _is_object_reference(testcase), "Case {} should return {}".format(testcase, result))
+
```
Comment (Vegter) [inline src/tests/test_converter.py]: I find
f"Case {testcase} should return {result}"
more readable

File: src/gobimport/converter.py [hunk 2]
```diff
@@ -72,6 +72,30 @@ def _is_literal(field):
     return field[0] == "="
 
 
+def _is_object_reference(field: str):
+    """
+    Returns whether the field references an object attribute
```
Comment (Vegter) [inline src/gobimport/converter.py]: Probably a real-world example could explain why this is used?

File: src/tests/test_converter.py (hunk 1 above)
Comment (JKoetsier) [reply src/tests/test_converter.py]: You're right, if only for consistency :) Changed

File: src/gobimport/converter.py (hunk 2 above)
Comment (JKoetsier) [reply src/gobimport/converter.py]: Check

**Pull request merged**